protocol-designer/src/components/FileSidebar/FileSidebar.tsx

```diff
@@ -376,11 +376,11 @@ export function FileSidebar(): JSX.Element {
             onCloseClick={cancelModal}
             buttons={[
               {
-                children: 'CANCEL',
+                children: t('Cancel'),
```

Comment:
Is there any specific reason to use upperCase for the `C`?

Reply:
Just saw the json file.
```suggestion
                children: t('cancel'),
```

Reply:
oops thank you! this was definitely a typo